Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3.6-slim
+
+MAINTAINER Schemen <me@schemen.me>
+
+EXPOSE 8000
+
+WORKDIR /usr/src/app
+
+VOLUME /usr/src/app/data
+VOLUME /usr/src/app/static/assets
+
+ENTRYPOINT ["/usr/bin/dumb-init", "--"]
+
+COPY requirements.txt ./
+RUN apt-get update && apt-get install dumb-init && \
```

Comment:
I think it's a good idea to do `rm -rf /var/lib/apt/lists/*` after the install completed. The docker container won't need the apt list anymore and this takes up a decent amount of space.

Reply:
Good Idea, done that.